data/brands/shop/department_store.json

```
@@ -1007,6 +1046,17 @@
         "name": "Nordstrom",
         "shop": "department_store"
       }
+    },
+     {
+      "displayName": "Nuance",
+      "id": "nuance-b0f55b",
+      "locationSet": {"include": ["eu","uk"]},
```

```suggestion
      "locationSet": {"include": ["001"]},
```
It appears they also have a location in Canada. (https://toronto.shopdutyfree.com/en/). I think it's best to include the whole world as they could have locations in other countries as well.